src/c++/perf_analyzer/genai-pa/genai_pa/parser.py

```diff
@@ -57,75 +65,116 @@ def add_model_args(parser):
     )
 
 
-def add_profile_args(parser):
+def add_profile_args(parser, exclusive_group):
     parser.add_argument(
         "-b",
-        "--batch-size",
         type=int,
         default=1,
         required=False,
-        help="The batch size / concurrency to benchmark. (Default: 1)",
+        help="The batch size to benchmark. The default value is 1.",
     )
-    parser.add_argument(
-        "--input-length",
+    exclusive_group.add_argument(
+        "--concurrency",
         type=int,
-        default=128,
         required=False,
-        help="The input length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the concurrency value to benchmark.",
     )
     parser.add_argument(
-        "--output-length",
+        "--max-threads",
         type=int,
-        default=128,
+        default=16,
         required=False,
-        help="The output length (tokens) to use for benchmarking LLMs. (Default: 128)",
+        help="Sets the maximum number of threads that will be "
+        "created for providing desired concurrency or request rate. "
+        "However, when running in synchronous mode,this value will be ignored. "
```

Comment:
Would remove mention of synchronous mode if going with an hard-coded default of async.
```suggestion
```